ui/imports/shared/popups/SendModal.qml

```diff
@@ -125,171 +127,165 @@ StatusDialog {
-                        validators: [
-                            StatusFloatValidator{
-                                id: floatValidator
-                                bottom: 0
-                                top: d.maxFiatBalance
-                                errorMessage: qsTr("Please enter a valid amount")
-                            }
-                        ]
-                        Keys.onReleased: {
-                            let amount = amountToSendInput.text.trim()
 
-                            if (!Utils.containsOnlyDigits(amount) || isNaN(amount)) {
-                                return
-                            }
-                            if (amount === "") {
-                                txtFiatBalance.text = "0.00"
-                            } else {
-                                txtFiatBalance.text = popup.store.getFiatValue(amount, assetSelector.selectedAsset.symbol, popup.store.currentCurrency)
-                            }
-                            gasSelector.estimateGas()
-                            popup.recalculateRoutesAndFees()
-                        }
-                    }
-                    StatusAssetSelector {
-                        id: assetSelector
-                        anchors.verticalCenter: parent.verticalCenter
-                        anchors.right: parent.right
-                        assets: popup.selectedAccount.assets
-                        defaultToken: Style.png("tokens/DEFAULT-TOKEN@3x")
-                        getCurrencyBalanceString: function (currencyBalance) {
-                            return Utils.toLocaleString(currencyBalance.toFixed(2), popup.store.locale, {"currency": true}) + " " + popup.store.currentCurrency.toUpperCase()
+
+                Column {
+                    id: assetAndAmmountSelector
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.leftMargin: Style.current.xlPadding
+                    anchors.rightMargin: Style.current.xlPadding
+                    z: 1
+
+                    Row {
+                        spacing: 16
+                        StatusBaseText {
+                            anchors.verticalCenter: parent.verticalCenter
+                            text: qsTr("Send")
+                            font.pixelSize: 15
+                            color: Theme.palette.directColor1
                         }
-                        tokenAssetSourceFn: function (symbol) {
-                            return symbol ? Style.png("tokens/" + symbol) : defaultToken
+                        StatusListItemTag {
+                            height: 22
+                            width: childrenRect.width
+                            title: assetSelector.selectedAsset.totalBalance > 0 ? qsTr("Max: ") + (assetSelector.selectedAsset ? d.maxFiatBalance : "0.00") : qsTr("No balances active")
```

Comment:
Looks a bit cleaner with the `arg` thingy.
```suggestion
                            title: assetSelector.selectedAsset.totalBalance > 0 ? qsTr("Max: %1") .arg(assetSelector.selectedAsset ? d.maxFiatBalance : "0.00") : qsTr("No balances active")
```

Reply:
Yes, and w/o the literal "0.00"